src/usecases/recipes.js

```diff
@@ -1,7 +1,14 @@
 const Recipes = require('../models/recipes')
+const User = require('../models/users')
 
-function getAll () {
-    return Recipes.find({})
+
+async function getByUser (idUser) {
+    if (idUser) {
```

Comment:
Si el idUser es null, es decir, que no venga en el request retornara todas la recetas va.

```suggestion
    if (!idUser) {
```